src/styles/_common.scss

```diff
@@ -1 +1,17 @@
-$card-background-color: #f5f5f5;
\ No newline at end of file
+$card-background-color: #f5f5f5;
+
+
+.cta--center--position{
+    margin-left: 40%;
```

Comment:
We have our own layout css for position the content inside the container. See my PR in the A-light/B-light.